### PR TITLE
Added programs.csv to GIDS

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As an EDU team member, I need the ability to upload program data to GIDS so that it can be used to display information in the Comparison Tool

## Testing done

- Open GIDS
- Navigate to the programs and click export
- Add a row in the programs.csv
- Upload the updated programs.csv in the GIDS tool
- Check the POSTGRE database to confirm changes
- Download export CSV and confirm changes

## Screenshots
Screen Shot 2019-08-13 at 10 32 52 AM
Screen Shot 2019-08-13 at 10 32 19 AM
Screen Shot 2019-08-13 at 10 31 00 AM

## Acceptance criteria
- [x] A user can upload a .csv file containing program data matching the format outlined in Supporting Artifact # 1 into the GIDS Tool.
- [x] A user can export the most recently uploaded program data from the GIDS Tool.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

File: app/models/programs.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Programs < ActiveRecord::Base
```
Comment (jholton) [inline app/models/programs.rb]: Following Rails conventions, this class (and file name) should be `Program` (singular).

Comment (ksantiagomorgan6) [reply app/models/programs.rb]: 10-4, let me fix that

Comment (ksantiagomorgan6) [reply app/models/programs.rb]: Updated it Mr. Holton

Comment (jholton) [reply app/models/programs.rb]: No...?  Still says "Programs" and file is still named "programs.rb"

File: spec/factories/program.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    facility_code { generate :facility_code }
+    institution_name { 'NORTHEASTERN ILLINOIS UNIVERSITY' }
+    program_type 'NCD'
+    description 'COMPUTER SCIENCE'
+    full_time_undergraduate '360'
+    graduate '234'
+    full_time_modifier 'S'
+    length '1001'
+
+    initialize_with do
```
Comment (jholton) [inline spec/factories/program.rb]: I think this `initialize_with` block is unnecessary.

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**